src/main/java/dataObjects/Data.java

```diff
@@ -0,0 +1,9 @@
+package dataObjects;
+
+import java.util.List;
+
+public class Data {
+    String labelType; // ["text", "line", "dot"...]
```

Comment:
Consider making this an enum value, that way we have a simple way to create a selection in the GUI that would change if we change the label type options.